tools/UnitTestApp/script.ts

```diff
@@ -1,24 +1,91 @@
 const fs = require('fs');
-const readFilePath = './node_modules/@fluentui/react-charting/lib-commonjs/components/Sparkline/Sparkline.base.js';
-const writeFilePath = './Sparkline/SparklineBase.js';
+const path = require('path');
+
+async function readDirectoryDeep(dirPath) {
+  const items = await fs.readdirSync(dirPath);
+
+  items.forEach(item => {
+      const fullPath = path.join(dirPath, item);
+      const stat = fs.statSync(fullPath);
+
+      if (stat.isDirectory()) {
+          readDirectoryDeep(fullPath);
+      } else {
+          updatedFileDeep(fullPath);
+      }
+  });
+}
+
 // Read the file
-const updatedFile = () => {
-  fs.readFile(readFilePath, 'utf8', (err, data) => {
-    if (err) {
-      console.error(`Error reading file from disk: ${err}`);
-    } else {
-      // Replace the words
-      let result = data.replace(/require\("d3-scale"\)/g, "import('d3-scale')");
-      result = result.replace(/require\("d3-shape"\)/g, "import('d3-shape')");
-      result = result.replace(/require\("d3-array"\)/g, "import('d3-array')");
-      // Write the file back
-      fs.writeFile(writeFilePath, result, 'utf8', err => {
-        if (err) {
-          console.error(`Error writing file to disk: ${err}`);
-        }
-      });
+async function updatedFileDeep(filePath) {
+  let data = await fs.readFileSync(filePath, 'utf8'); 
+  let matches = data.match(/.*=.*d3.*(\n.*?)*(?=var|$)/g);
+  if (matches != null && matches.length > 0) {
+    matches.forEach(match => {
+      if (match !=null) {
+        data = data.replace(match, '');
+      }
+    });
+    // Write the file back
+    if (data != null && data != '' && data != undefined) {
+      await fs.writeFileSync(filePath, data);
     }
+  }
+};
+
+async function readDirectory(dirPath) {
```

Comment:
which function is to be used readDirectory or readDirectoryDeep?

Reply:
both for different purposes, for some files like Utility, only shallow updation is enough. But for base js modules, we need deep updation.

Reply:
add comments for it if not done already.